lib/terminal.robot

```diff
@@ -160,3 +160,13 @@ Execute Command In Terminal
     # Drop last newline, if any
     ${output}=    Strip String    ${output}    mode=right    characters=\n\r
     RETURN    ${output}
+
+Execute Shell Command
+    [Documentation]    Universal keyword to execute command in Shell.
```

Comment:
Is it really universal, or meant for UEFI shell? If meant for UEFI shell, I would rename the keyword to reflect that

Reply:
changed name in: 0a78cd0aa92831ffd253a3f5d86ff71def8bcde8